src/extendedstorage.cpp

```diff
@@ -129,6 +113,16 @@ class mKCal::ExtendedStorage::Private
     void commitEvents(Timed::Event::List &events);
 #endif
 };
+
+bool ExtendedStorage::Private::close()
```

Comment:
We cannot call the public `close()` since this public method is virtual and we don't want to actually close the database access as done in `SqliteStorage::close()`. Which imply that the name is badly chosen, I you said. So I'm keeping this routine and simply changing its name to `clear()`.